### PR TITLE
dolt-fetch: add page

### PR DESCRIPTION
**Pull request by Torfab**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by Torfab** [comment]
PR for issue #5471

File: pages/common/dolt-fetch.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# dolt fetch
+
+> Download objects and refs from another repository.
+> More information: <https://github.com/dolthub/dolt>.
+
+- Fetch refs and update remote-tracking branches from the default remote named "origin":
+
+`dolt fetch`
+
+- Fetch refs and update remote-tracking branches from a specific remote:
```
Comment (navarroaxel) [inline pages/common/dolt-fetch.md:10]: Perhaps this better
```suggestion
- Fetch latest changes from a specific remote upstream repository:
```

Comment (Torfab) [reply pages/common/dolt-fetch.md]: solved in c75c5d817b331439b6fa3e264d6b0baa96923792

File: pages/common/dolt-fetch.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# dolt fetch
+
+> Download objects and refs from another repository.
+> More information: <https://github.com/dolthub/dolt>.
+
+- Fetch refs and update remote-tracking branches from the default remote named "origin":
```
Comment (navarroaxel) [inline pages/common/dolt-fetch.md:6]:
```suggestion
- Fetch the latest changes from the default remote upstream repository (origin):
```

File: pages/common/dolt-fetch.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+`dolt fetch`
+
+- Fetch latest changes from a specific remote upstream repository:
+
+`dolt fetch {{remote_name}}`
+
+- Update refs to remote branches with the current state of the remote, overwriting any conflicting history:
```
Comment (navarroaxel) [inline pages/common/dolt-fetch.md:14]: Perhaps this better
```suggestion
- Update branches with the current state of the remote, overwriting any conflicting history:
```

File: pages/common/dolt-fetch.md (hunk 2 above)
Comment (Torfab) [reply pages/common/dolt-fetch.md]: solved in 95bb1b6bf7274caad847319db358b771a1736bcf

File: pages/common/dolt-fetch.md (hunk 3 above)
Comment (Torfab) [reply pages/common/dolt-fetch.md]: solved in 95bb1b6bf7274caad847319db358b771a1736bcf

**Pull request merged**